src/confluent_kafka/admin/__init__.py

```diff
@@ -522,6 +523,35 @@ def alter_configs(self, resources, **kwargs):
 
         return futmap
 
+    def incremental_alter_configs(self, resources, **kwargs):
+        """
+        Update configuration properties for the specified resources.
+        Updates are incremental, i.e only the values mentioned are changed
+        and rest remain as is.
+
+        :param list(ConfigResource) resources: Resources to update configuration of.
```

Comment:
Instead of `list(ConfigResource)`, can we create a class `IncrementalAlterConfigsRequest` having this list as a field and take that as input. So that its extensible in future if required.

Reply:
In J. it cannot be extended, as they have a Map currently. If there's need to extend it they'll create a new API too.